regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -72,8 +72,11 @@ var CommentReviewView = Backbone.View.extend({
   },
 
   serialize: function() {
+    // TODO(vrajmohan) Add other regs.gov fields
     return {
-      sections: comments.toJSON({docId: this.docId})
+        general_comment:  {
```

Comment:
Good point. I did not realize that it was this confusing when I wrote it. I do see that I was _abusing_ the `general_comment` field.